apps/showcase/src/assets/trainings/sdk/steps/sdk-generation/instructions.html

```diff
@@ -0,0 +1,77 @@
+<p>
+  There are several command lines available to generate an SDK depending on the use cases. These cases are covered in the
+  <a href="https://github.com/AmadeusITGroup/otter/tree/main/packages/%40ama-sdk/schematics" target="_blank">documentation</a>
+  of the package <code>@ama-sdk/schematics</code>.
+</p>
+
+<h3>Objective</h3>
+<p>
+  Generate a new SDK repository using the previous specification file (Swagger Petstore) and the provided command line.
+</p>
+
+<h3>Exercise</h3>
+<h4>Generation of the SDK repository</h4>
+<p>
+  To generate a new SDK repository, you can run the following command line (by replacing <code>&lt;package-name&gt;</code> with <code>training-sdk</code> for example):
+</p>
+<pre class="w-100 bg-body-tertiary px-5">
+  <code>
+npm create @ama-sdk typescript &lt;package-name&gt;
+  </code>
+</pre>
+<p>
+  You will receive a prompt asking you to specify the project name. For example, you can set the project name to <code>training-project</code>.
+</p>
+<pre class="w-100 bg-body-tertiary px-5 pre-whitespace">
+  <code>
+<span class="text-success">?</span> Project name (NPM package scope, package.json name will be &#64;&#123;projectName&#125;/&#123;packageName&#125;)? <span class="text-primary">training-project</span>
+  </code>
+</pre>
+<p>
+  After setting the project name, you will see the shell of your SDK repository being generated with a structure that looks something like the following:
+</p>
+<ul>
+  <li><span class="fa-folder-open"></span> src
+    <ul>
+      <li><span class="fa-folder"></span> api</li>
+      <li><span class="fa-folder"></span> fixtures</li>
+      <li><span class="fa-folder"></span> helpers</li>
+      <li><span class="fa-folder-open"></span> models
+        <ul>
+          <li><span class="fa-folder"></span> base</li>
+          <li><span class="fa-folder"></span> core</li>
+          <li><span class="fa-folder"></span> custom</li>
+        </ul>
+      </li>
+      <li><span class="fa-folder"></span> spec</li>
+    </ul>
+  </li>
+</ul>
+<p>
+  You should also observe these scripts in the <code>package.json</code> file (with the correct project and package names):
+</p>
+<pre class="w-100 bg-body-tertiary px-5">
```

Comment:
I assume we can only put HTML here, no way to have code highlighting ?

Reply:
indeed, we can only use HTML in these files